kuzzleio/setup-script
# Use Shields.io API instead of versioned SVG file

## What new?
Instead of adding raw SVG file to Git Tree use shields.io API to generate badges.

## How to test it?

Go to https://github.com/kuzzleio/setup-script/tree/update-badges and check the badge is displayed correctly.
